ompi/dpm/dpm.c

```diff
@@ -1193,10 +1221,11 @@ int ompi_dpm_spawn(int count, const char *array_of_commands[],
                 }
                 continue;
             }
-            ompi_info_get (array_of_info[i], "pernode", sizeof(slot_list) - 1, slot_list, &flag);
+            ompi_info_get (array_of_info[i], "pernode", &info_str, &flag);
             if ( flag ) {
                 rc = dpm_convert(&job_info, "pernode", PMIX_MAPBY, "PPR:1:NODE", NULL, true);
                 free(tmp);
+                OBJ_RELEASE(info_str);
```

Comment:
This is a side effect of this PR: is this a bug?  We get the "pernode" info value, but then it isn't used -- instead, it looks like a hard-coded `PPR:1:NODE` string is used.  

@rhc54 Is that correct?

Reply:
Yeah - the "pernode" flag is a deprecate info key specifying the launch of 1 proc/node and has no arguments. It is therefore translated directly to "ppr:1:node".